src/server/utils/configSchema.ts

```diff
@@ -72,30 +62,9 @@ export const configSchema = Joi.object({
   fromEmailAddress: Joi.string().optional().allow(''),
   serviceStartPage: Joi.string().optional().allow(''),
   privacyPolicyUrl: Joi.string().optional().allow(''),
-  notifyTemplateId: Joi.string().optional().allow(''),
-  notifyAPIKey: Joi.string().optional().allow(''),
+  notifyTemplateId: Joi.string().required(),
+  notifyAPIKey: Joi.string().required(),
```

Comment:
Now that they're required, you'll need to add both of these into:

**config.test.js**
```
const requiredVars = {
    // the others
    notifyTemplateId: "dummy",
    notifyAPIKey: "dummy"
}
```

**jest.setup.cjs**
```
process.env.NOTIFY_TEMPLATE_ID = "dummy"
process.env.NOTIFY_API_KEY = "dummy"
```

Reply:
Resolved wirth 7c80a6f234a2f0a69c6ee39b6862528b6aa53ab5